src/Collection/Collection.php

```diff
@@ -63,22 +63,6 @@ public function __unserialize(array $data): void
         $this->__construct($data);
     }
 
-    /**
-     * {@inheritDoc}
-     *
-     * @return int
-     */
-    public function count(): int
```

Comment:
Do we have another count method in the trait?

Reply:
Yup, CollectionTrait already has one.